week-4/Homework/mandatory/2-writers.js

```diff
@@ -54,18 +56,35 @@ Exercise 1:
   "Hi, my name is {firstName} {lastName}. I am {age} years old, and work as a {occupation}."
 */
 
-
+writers.forEach((p) => {
+  console.log(
+    `Hi, my name is ${p.firstName}${p.lastName}. I am ${p.age} years old, and work as a ${p.occupation}.`
+  );
+});
 
 /*
 Exercise 2:
   Only `console.log()` the writers who are in their 40s with us anymore with the sentence:(meaning between 40 and 49)
   "Writer {firstName} {lastName} died at {age} years old."
 */
 
-
+const writersDead = writers
+  .filter((x) => x.age > 40 && x.age < 50 && x.alive === false)
+  .forEach((x) => {
+    console.log(
+      `Writer ${x.firstName} ${x.lastName} died at ${x.age} years old.`
+    );
+  });
 
 /*
 Exercise 3:
   Only `console.log()` contemporary writers who are in their forties:
   "Hi, my name is {firstName} {lastName}. I am {age} years old."
 */
+const comtemWriters = writers
+  .filter((x) => x.age >= 40 && x.age <= 49)
+  .forEach((x) => {
+    console.log(
+      `Hi, my name is ${x.firstName} ${x.lastName}. I am ${x.age} years old.`
+    );
+  });

```

Comment:
You need to filter out dead writers, that is, exclude Jane Austen.

Reply:
done